### ISSUE TITLE
Quotes make alias lookup fail

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#131]
```
What steps will reproduce the problem?
1. Type: sms:it's

What is the expected output?
Either: No match for "it's"
or a list of sms'es for a contact matching that term.

What do you see instead?
Error: android.database.sqlite.SQLiteException: near "s": syntax error: , while 
compiling: SELECT number FROM alias WHERE aliasName='it's'

Additional informations:
GTalkSMS version = 2.1
Android OS version = 2.2
Phone model = HTC Hero

```

Original issue reported on code.google.com by `Eckankar` on 24 Mar 2011 at 11:50

**Comment by GoogleCodeExporter** [comment]
```
doh, thanks for reporting
```

Original comment by `fschm...@gmail.com` on 24 Mar 2011 at 5:54
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Mar 2011 at 5:37
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 2.1.1
```

Original comment by `Florent....@gmail.com` on 27 Mar 2011 at 3:08
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:50
- Changed state: **Deployed**
